larq_compute_engine/mlir/python/graphdef_tfl_flatbuffer.cc

```diff
@@ -118,11 +107,8 @@ pybind11::bytes ConvertGraphDefToTFLiteFlatBuffer(
   pm.addPass(mlir::TFL::CreateRuntimeVerifyPass());
 
   std::string result;
-  auto status = ConvertTFExecutorToTFLOrFlatbuffer(
-      module->get(), /*export_to_mlir=*/false, /*emit_builtin_tflite_ops=*/true,
-      /*emit_select_tf_ops=*/false, /*emit_custom_ops=*/true,
-      /*select_user_tf_ops=*/{}, quant_specs, /*saved_model_tags=*/{}, &result,
-      &pm);
+  auto status = ConvertTFExecutorToFlatbuffer(
+      module->get(), /*export_to_mlir=*/false, &result, &pm);
```

Comment:
I kept the `export_to_mlir` option since it might come in handy for debugging later on.

Reply:
That's amazing, I'm always searching for that option when I want to write new Filecheck tests.